frontend/src/pages/profile/components/user-achievements/styles.module.scss

```diff
@@ -28,6 +32,7 @@
 }
 
 .modal {
+  overflow: hidden;
```

Comment:
Do we need this? The Modal component has it's style `overflow: auto;`

Reply:
In this case it is crucial for us that tooltip should be located within the modal window.
This is required for tooltip-position-strategy
https://react-tooltip.com/docs/options

Examples:

![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/136931364/094365ae-8d04-44d6-94a5-bc51e488a750)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/assets/136931364/bcf41fcf-7e9a-44e0-a468-d67900978bb1)

Reply:
Okay, I got it. Thanks!